api/poi.go

```diff
@@ -287,13 +288,7 @@ func (s *Server) listPOI(c *gin.Context) {
 	}
 
 	sort.SliceStable(response, func(i, j int) bool {
-		if response[i].ResourceScore == nil {
-			return false
-		}
-		if response[j].ResourceScore == nil {
-			return true
-		}
-		return *response[i].ResourceScore > *response[j].ResourceScore
+		return response[i].ResourceScore > response[j].ResourceScore
```

Comment:
`POIDetail.ResourceScore` is still a pointer, and the the operator `>` can't be applied on it.

Reply:
@nodestory It is not. The type has been modified in #107.